make_pao1_pa14_compendia/0_decide_threshold.py

```diff
@@ -0,0 +1,215 @@
+)
+
+# +
+# Aggregate boolean labels into a single strain label
+aggregated_label = []
+for exp_id in list(sample_to_strain_table_full_processed.index):
+    if sample_to_strain_table_full_processed.loc[exp_id, "PAO1"].all() == True:
+        aggregated_label.append("PAO1")
+    elif sample_to_strain_table_full_processed.loc[exp_id, "PA14"].all() == True:
+        aggregated_label.append("PA14")
+    elif sample_to_strain_table_full_processed.loc[exp_id, "PAK"].all() == True:
+        aggregated_label.append("PAK")
+    elif (
+        sample_to_strain_table_full_processed.loc[exp_id, "ClinicalIsolate"].all()
+        == True
+    ):
+        aggregated_label.append("Clinical Isolate")
+    else:
+        aggregated_label.append("NA")
+
+sample_to_strain_table_full_processed["Strain type"] = aggregated_label
+
+sample_to_strain_table = sample_to_strain_table_full_processed["Strain type"].to_frame()
+
+sample_to_strain_table.head()
+# -
+
+# ## Label samples with SRA annotations
+
+# Label samples with SRA annotations
+# pao1_expression_label = pao1_expression_binned.join(
+#    sample_to_strain_table, how='left')
+pao1_expression_label = pao1_expression.merge(
+    sample_to_strain_table, left_index=True, right_index=True
+)
+pa14_expression_label = pa14_expression.merge(
+    sample_to_strain_table, left_index=True, right_index=True
+)
+print(pao1_expression_label.shape)
+pao1_expression_label.head()
+
+print(pa14_expression_label.shape)
+pa14_expression_label.head()
+
+# ## Get accessory gene expression
+
+# +
+pao1_annot_filename = "PAO1_ID_2_PA14_ID_PAO1ref.csv" 
+pa14_annot_filename = "PA14_ID_2_PAO1_ID_PA14ref.csv"
+
+core_acc_dict = utils.get_my_core_acc_genes(
+    pao1_annot_filename, pa14_annot_filename, pao1_expression, pa14_expression
+)
+# -
+
+pao1_acc = core_acc_dict["acc_pao1"]
+pa14_acc = core_acc_dict["acc_pa14"]
+
+# +
+# Create accessory df
+# accessory gene ids | median accessory expression | strain label
+
+# PAO1
+pao1_acc_expression = pao1_expression_label[pao1_acc]
+pao1_acc_expression["median_acc_expression"] = pao1_acc_expression.median(axis=1)
+
+# PA14
+pa14_acc_expression = pa14_expression_label[pa14_acc]
+pa14_acc_expression["median_acc_expression"] = pa14_acc_expression.median(axis=1)
+# -
+
+# Add back labels
+pao1_acc_expression["Strain type"] = pao1_expression_label["Strain type"]
+pa14_acc_expression["Strain type"] = pa14_expression_label["Strain type"]
+
+pao1_acc_expression.head()
+
+pa14_acc_expression.head()
+
+# +
+# Merge PAO1 and PA14 accessory dataframes
+pao1_pa14_acc_expression = pao1_acc_expression.merge(
+    pa14_acc_expression,
+    left_index=True,
+    right_index=True,
+    suffixes=["_pao1", "_pa14"],
+)
+
+pao1_pa14_acc_expression.head()
+# -
+
+# ## Plot distribution
+
+# +
+# Get PAO1 samples that are labeled PAO1 and non-PAO1
+pao1_sra = pao1_pa14_acc_expression.loc[
+    pao1_pa14_acc_expression["Strain type_pao1"] == "PAO1",
+    "median_acc_expression_pao1",
```

Comment:
should these labels maybe read "median normalized accessory gene expression" I know that is kind of long, but i think it would be good to have the fact that it is normalized counts indicated since it could have been done with raw counts or TPM... maybe "median pao1 accessory expression (normalized counts)" ?